kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -70,6 +75,20 @@ inline const std::string getCategoryType(const std::variant<BlockMerkleInput, Ve
       c);
 }
 
+template <typename IterableStringPairs>
+std::string toJson(const IterableStringPairs &container) {
+  std::ostringstream oss;
+  oss << "{";
+  for (const auto &it : container) {
+    oss << "\n\t\"" << it.first << "\":\"" << it.second << "\",";
+  }
+  if (!container.empty()) {
+    oss << "\n";
+  }
+  oss << "}";
+  return oss.str();
+}
+
```

Comment:
I think there is already a similar function in json_output.hpp:

```
template <typename KVContainer, typename Encoder>
inline std::string kvContainerToJson(const KVContainer &kv, const Encoder &enc)
```

Can we use it?